base/operators.jl

```diff
@@ -5,8 +5,26 @@
 """
     <:(T1, T2)
 
-Subtype operator: returns `true` if and only if all values of type `T1` are
-also of type `T2`.
+Subtyping relation, defined between two types. The returned value is of type
+`Bool`.
```

Comment:
style nit: we do not write this in text, but may put it as a return type annotation on the signature above if relevant
```suggestion
Subtyping relation, defined between two types
```